### PR TITLE
Down the White Nile: NGO Front

### PR DESCRIPTION
**Pull request by butzopower**
Quite an easy one to implement :)

### CODE REVIEW

**Comment by butzopower** [comment]
#3025

File: src/clj/game/cards/assets.clj [hunk 1]
```diff
@@ -979,6 +979,17 @@
                                                    (trigger-event state side :no-trash card))))}
                                card targets))}}
 
+   "NGO Front"
+   {:advanceable :always
+    :abilities [{:advance-counter-cost 1
+                 :effect (effect (trash card) (gain :credit 5))
```
Comment (nealterrell) [inline src/clj/game/cards/assets.clj]: Add a final parameter `{:cause :ability-cost}` to the `trash` calls to indicate that we're paying an ability cost as the reason for the trash.

File: src/clj/game/cards/assets.clj [hunk 2]
```diff
@@ -979,6 +979,17 @@
...
+                 :label "Gain 5 [Credits]"
```
Comment (nealterrell) [inline src/clj/game/cards/assets.clj]: Let's put `[Trash]` at the beginning of the label string, to show the trashcan icon in the menu item itself: `"[Trash]: Gain 5 [Credits]"`.

Comment (Saintis) [reply src/clj/game/cards/assets.clj]: I'd be very tempted to make both these abilities into an ability-maker function that takes advance-counter-cost and credit-gain to reduce the minor repetition.

**Comment by butzopower** [comment]
Thanks for all the feedback @nealterrell and @Saintis, updated my commit.

**Pull request merged**